public/app/features/alerting/unified/api/alertmanagerApi.ts

```diff
@@ -226,5 +227,30 @@ export const alertmanagerApi = alertingApi.injectEndpoints({
       }),
       invalidatesTags: ['AlertmanagerConfiguration'],
     }),
+
+    // Grafana Managed Alertmanager only
+    getContactPointsStatus: build.query<ReceiversStateDTO[], void>({
+      query: () => ({
+        url: `/api/alertmanager/${getDatasourceAPIUid(GRAFANA_RULES_SOURCE_NAME)}/config/api/v1/receivers`,
+      }),
+      // this transformer basically fixes the weird "0001-01-01T00:00:00.000Z" timestamps
+      // and sets both last attempt and duration to an empty string to indicate there hasn't been an attempt yet
+      transformResponse: (response: ReceiversStateDTO[]) => {
+        const isLastNotifyNullDate = (lastNotify: string) => lastNotify === '0001-01-01T00:00:00.000Z';
```

Comment:
I've seen the API return this null date as `0001-01-01T00:00:00.000Z` but also as `0001-01-01T00:00:00Z`  in different responses. Maybe we could add a check for that last one as well or just compare the first section?

Reply:
Ah good point, probably depends on the granularity of the timestamp, will update